x-pack/spec/filters/geoip/test_helper.rb

```diff
@@ -6,7 +6,7 @@
 require "digest"
 
 def get_vendor_path
-  ::File.expand_path("./vendor/", ::File.dirname(__FILE__))
```

Comment:
These helpers should be defined in a [self-contained module](https://relishapp.com/rspec/rspec-core/docs/helper-methods/define-helper-methods-in-a-module), and included into the relevant specs. Otherwise we pollute `Object` (and therefore _all_ objects):

~~~ ruby
irb(main):001:0> def test; :oh_no; end;
irb(main):002:0* Object.new.test
=> :oh_no
irb(main):003:0> nil.test
=> :oh_no
irb(main):004:0> true.test
=> :oh_no
irb(main):005:0> 1.test
=> :oh_no
irb(main):006:0> "hello".test
=> :oh_no
~~~

The risk here is that we create a situation where our helpers in specs can accidentally provide the methods that they are attempting to specify, leaving the actual methods untested.

Reply:
:sob: :oh_no